internal/logs/fetcher_test.go

```diff
@@ -63,6 +62,32 @@ func Test(t *testing.T) {
 	}
 }
 
+func Test_stripANSI(t *testing.T) {
+	testCases := []struct {
```

Comment:
Could you please add `t.Parallel()` as the first line of the test so that it runs concurrently to others?
This reduces test execution time significantly.

Reply:
Done!